package.json

```diff
@@ -1,60 +1,49 @@
 {
-  "nativescript": {
-    "id": "org.nativescript.n65sample",
-    "tns-android": {
-      "version": "6.5.3"
-    },
-    "tns-ios": {
-      "version": "6.5.3"
-    }
-  },
   "scripts": {
-    "clean": "npx rimraf hooks node_modules package-lock.json platforms",
+    "clean": "npx rimraf hooks node_modules package-lock.json platforms && npm i",
     "ng": "ng",
     "start": "ng serve",
     "build": "ng build",
     "test": "ng test",
     "lint": "ng lint",
     "e2e": "ng e2e",
-    "android": "tns run android --no-hmr",
-    "ios": "tns run ios --no-hmr",
-    "mobile": "tns run",
-    "preview": "tns preview",
-    "ngcc": "ngcc --properties es2015 module main --first-only",
-    "postinstall": "npm run ngcc"
+    "android": "ns debug android --no-hmr",
+    "ios": "ns debug ios --no-hmr",
```

Comment:
Who runs an app that doesn’t actively debug it? Are you running to show on a TV 📺 ;) in all seriousness debug should be what run does by default - do you run a web app with debug tools disabled?

hmr has some improvements to be made so we recommend not using it at moment although in some cases it may work fine.